docs/source/item-validation.rst

```diff
@@ -227,7 +229,7 @@ Some examples:
     self.check_missing_required_fields(field_names=['field2', 'field3'], allowed_count=10)
 
     # checks that field2 has errors in no more than 15% of items
-    self.check_field_errors_percent(field_name='field2', allowed_percent=15)
+    self.check_field_errors_percent(field_name='field2', allowed_percent=0.15)
```

Comment:
The confusion is understandable given the parameter name. It may be worth opening an issue to rename it.

Reply:
It's also in the method names and deeper, it probably starts at https://github.com/scrapinghub/spidermon/blob/master/spidermon/contrib/stats/counters.py#L14 which these methods use.

Reply:
I am not against to have an issue to discuss better names for this method. We just need to be careful to avoid backward incompatible changes.